geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -38,6 +39,114 @@ app.module.directive('appMymaps', app.mymapsDirective);
  * @constructor
  * @export
```

Comment:
I'd recommend adding `@ngInject`. It's not necessary right now, but it will avoid some headache to the developer injecting services in the future.